src/dags/customnodesession.cpp

```diff
@@ -51,81 +51,6 @@ static std::unordered_map<std::string, shape_t> createOwnedShapesCopy(const Tens
     return tensorsDims;
 }
 
-Status CustomNodeSession::execute(PipelineEventQueue& notifyEndQueue, Node& node, const NodeLibrary& library, std::unique_ptr<struct CustomNodeParam[]>& parameters, int parametersCount, void* customNodeLibraryInternalManager) {
```

Comment:
Why do we need to move it to header?

Reply:
This was windows compilation issue. The method was not visible on windows compiler.

Reply:
Yes, I know this is windows compilation issue based off where this task comes from, but I'm just worried it will increase build time since we populate header with bunch of includes @atobiszei